app/src/main/java/com/example/packpals/repositories/PhotoRepository.kt

```diff
@@ -0,0 +1,64 @@
+package com.example.packpals.repositories
+
+import com.example.packpals.models.PhotoAlbum
+import com.google.firebase.firestore.CollectionReference
+import com.google.firebase.storage.FirebaseStorage
+import com.google.firebase.storage.StorageReference
+
+import kotlinx.coroutines.tasks.await
+import javax.inject.Inject
+
+class PhotoRepository @Inject constructor (private val photoCollectionRef: CollectionReference) {
+
+    private val storage = FirebaseStorage.getInstance()
```

Comment:
probably just use StorageRepository for any firebase cloud storage calls and add any functions you need there instead

Reply:
kk